docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -212,12 +217,33 @@ Iceberg. In addition to the :ref:`globally available <sql-globally-available>`
 and :ref:`read operation <sql-read-operations>` statements, the connector
 supports the following features:
 
+* :doc:`/sql/create-schema`, see also :ref:`iceberg-create-schema`
```

Comment:
We do NOT want to link to individual commands.. instead whenever possible it should link to the collection of command similar to read operations in the paragraph above.

See https://trino.io/docs/current/language/sql-support.html

And for example - https://trino.io/docs/current/language/sql-support.html#materialized-view-management

Each of these sections has anchors ..

We only link to individual SQL commands if not all commands in a section are supported (and that should actually not be the case and is considered a bug/feature request)